### ISSUE TITLE
Where is unsafeCocoaAction?

### ISSUE THREAD

**Issue by sarunw** [ReactiveCocoa/ReactiveCocoa#3364]
I try to upgrade to RAC5, but `unsafeCocoaAction` is gone. What is the proper way to do unsafeCocoaAction right now?

**Comment by andersio** [comment]
You can use the initializer.

**Comment by sarunw** [comment]
You mean create it manually as instance variable in vc/vm?

**Comment by andersio** [comment]
`action.unsafeCocoaAction` and `CocoaAction(action)` have no difference at all for `Action` that has an input type of `()`. If you are using binding APIs like `reactive.pressed`, you can simply do:

```swift
// `action` is `Action<(), _, _>`.
button.reactive.pressed = CocoaAction(action)

// `action` is `Action<Bool, _, _>`.
switch.reactive.toggled = CocoaAction(action) { switch in switch.isOn }
```

**Comment by andersio** [comment]
Resolved by #3372.
